Packs/CyberArkPAS/Integrations/CyberArkPAS/CyberArkPAS_test.py

```diff
@@ -487,3 +487,27 @@ def test_pas_safe_members_list(mocker, response: dict, count_or_total: str, expe
     results = list_safe_members_command(client=client, safe_name="Test")
     assert count_or_total in results.readable_output
     assert results.outputs == expected_output
+
+
+@pytest.mark.parametrize("properties,expected", [
+    ({}, {}),
+    ({"a": 1}, {"a": 1}),
+    ({'a': 1}, {"a": 1}),
+    ("{'a': 1}", {"a": 1}),
+    ('{"a": 1}', {"a": 1}),
+    ("not json", ValueError)
+])
+def test_order_properties_to_dict(properties, expected):
+    """
+    Given:
+        - properties as input
+    When:
+        - order_properties_to_dict is called on properties
+    Then:
+        - it returns the expected output
+    """
+    if not isinstance(expected, dict):
+        with pytest.raises(expected):
+            order_properties_to_dict(properties)
+    else:
+        assert order_properties_to_dict(properties) == expected
```

Comment:
Split the test to a positive one (checking successful runs) and a negative (`pytest.raises`)